validator/keymanager/direct_keystore.go

```diff
@@ -45,6 +45,7 @@ func NewKeystore(input string) (KeyManager, string, error) {
 	if opts.Path == "" {
 		opts.Path = accounts.DefaultValidatorDir()
 	}
+	log.WithField("path", opts.Path).Info("Checking validator keystore path")
```

Comment:
Same here, can you change the text to "Checking validator keys" and the field to "keystorePath"?

Reply:
Okay, done